tests/10-verify.spec.js

```diff
@@ -379,15 +382,24 @@ describe('tests for multiple credentials', async () => {
       result.verified.should.be.false;
       credentialOne.verified.should.be.a('boolean');
       credentialOne.verified.should.be.false;
+
```

Comment:
Can you assert about the `credentiaId` for these since you know exactly what the ID is?